Added script to README.md for manual install

StooC opened the pull request:
I created a script for updating my HA install and thought it might be of use to the project. I've added it to the readme to save people needing to clone the repository just to get the script which will download the zip file but I could be add it to the repository with a reference in the readme if you prefer.

Bananahouse165 commented:
In the setup it says Edit startup.sh and add the following before the line # Start the ssh server, where is startup.sh?

StooC commented:
I just added a some instructions to the readme file instead of creating any scripts to save cloning the repo. The instructions I added don't mention setup.sh so not sure what you mean, could you give me a line number, please?
